hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/testutils/HiveTestUtil.java

```diff
@@ -154,6 +155,7 @@ public static void setUp() throws IOException, InterruptedException, HiveExcepti
     hiveSyncProps.setProperty(META_SYNC_PARTITION_FIELDS.key(), "datestr");
     hiveSyncProps.setProperty(META_SYNC_PARTITION_EXTRACTOR_CLASS.key(), SlashEncodedDayPartitionValueExtractor.class.getName());
     hiveSyncProps.setProperty(HIVE_BATCH_SYNC_PARTITION_NUM.key(), "3");
+    hiveSyncProps.setProperty(HIVE_AUTO_CREATE_DATABASE.key(), "true");
 
```

Comment:
This looks a big change for meta sync in high-level.

Reply:
yes, usually, there will be no problems with the default creat database, because user usually will create database first. I will close this pr if it is not need to change it.
and in out case, we use `waggle_dance` to router multi metastore instances, and there is something wrong in `waggle_dance` which can not correct connect the target ms instance when client check db is exist.
for example: 
we have two ms instances, is `A` and `B`
and db name `test` create in ms `A`, so every op in db `test` will all routing to this ms instance `A`.
but something wrong in `waggle_dance` happend, db check use the `B` instance, and will create a new db `test`, the db is not unique.
may be it has bug in `waggle_dance`, and I think `create db` seem to be unreasonable in each etl task

Reply:
Got it, let's keep this PR suspended for a while to enlighten with more ideas.